clang/test/CIR/driver.c

```diff
@@ -4,14 +4,14 @@
 // RUN: FileCheck --input-file=%t2.cir %s -check-prefix=CIR
 // RUN: %clang -target x86_64-unknown-linux-gnu -fclangir -fclangir-direct-lowering -S -emit-llvm %s -o %t1.ll
 // RUN: FileCheck --input-file=%t1.ll %s -check-prefix=LLVM
-// RUN: %clang -target x86_64-unknown-linux-gnu -fclangir -fno-clangir-direct-lowering -S -emit-llvm %s -o %t2.ll
-// RUN: FileCheck --input-file=%t2.ll %s -check-prefix=LLVM
+// RUN %clang -target x86_64-unknown-linux-gnu -fclangir -fno-clangir-direct-lowering -S -emit-llvm %s -o %t2.ll
+// RUN FileCheck --input-file=%t2.ll %s -check-prefix=CIR_STD_LLVM
```

Comment:
You've removed the `:`, so these won't run?

Reply:
Ouch, sorry, I removed the ':' for some internal checks to skip the run and I committed it by error. :-|